### PR TITLE
Fix line modification bugs

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/modifications/LineModification.java [hunk 1]
```diff
@@ -99,61 +97,91 @@ private void modifyCharacteristics(Line line, LineModificationInfos lineModifica
...
             .withSeverity(TypedValue.INFO_SEVERITY)
             .build());
-        if (lineModificationInfos.getSeriesResistance() != null) {
-            characteristicsReporter.report(ModificationUtils.getInstance().buildModificationReportWithIndentation(line.getR(),
-                    lineModificationInfos.getSeriesResistance().getValue(), "Series resistance", 1));
-            line.setR(lineModificationInfos.getSeriesResistance().getValue());
+        Double seriesResistance = lineModificationInfos.getSeriesResistance() != null
+                ? lineModificationInfos.getSeriesResistance().getValue()
+                : null;
+        if (seriesResistance != null) {
```
Comment (YenguiSeddik) [inline src/main/java/org/gridsuite/modification/server/modifications/LineModification.java:103]: We can check directly here: if (lineModificationInfos.getSeriesResistance() != null)

File: src/main/java/org/gridsuite/modification/server/modifications/LineModification.java [hunk 2]
```diff
@@ -99,61 +97,91 @@ private void modifyCharacteristics(Line line, LineModificationInfos lineModifica
...
         }
-        if (lineModificationInfos.getSeriesReactance() != null) {
-            characteristicsReporter.report(ModificationUtils.getInstance().buildModificationReportWithIndentation(line.getX(),
-                    lineModificationInfos.getSeriesReactance().getValue(), "Series reactance", 1));
-            line.setX(lineModificationInfos.getSeriesReactance().getValue());
+
+        Double seriesReactance = lineModificationInfos.getSeriesReactance() != null
+                ? lineModificationInfos.getSeriesReactance().getValue()
+                : null;
+        if (seriesReactance != null) {
```
Comment (YenguiSeddik) [inline src/main/java/org/gridsuite/modification/server/modifications/LineModification.java:113]: Same here

File: src/main/java/org/gridsuite/modification/server/modifications/LineModification.java [hunk 3]
```diff
@@ -99,61 +97,91 @@ private void modifyCharacteristics(Line line, LineModificationInfos lineModifica
...
+    private void modifySide1Characteristics(Line line, LineModificationInfos lineModificationInfos,
+            Reporter characteristicsReporter) {
+        Double shuntConductance1 = lineModificationInfos.getShuntConductance1() != null
+                ? lineModificationInfos.getShuntConductance1().getValue()
+                : null;
+        Double shuntSusceptance1 = lineModificationInfos.getShuntSusceptance1() != null
+                ? lineModificationInfos.getShuntSusceptance1().getValue()
+                : null;
+        if (shuntConductance1 != null
+                || shuntSusceptance1 != null) {
```
Comment (YenguiSeddik) [inline src/main/java/org/gridsuite/modification/server/modifications/LineModification.java:134]: i think we can check them one by one:
if (lineModificationInfos.getShuntConductance1() != null)
{......}
if (lineModificationInfos.getShuntSusceptance1() != null)
{......}

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Review by YenguiSeddik — APPROVED** [review]

**Pull request merged**